src/scenes/Root/Header/ProfileToolbar/ProfileToolbar.tsx

```diff
@@ -30,13 +23,13 @@ export const ProfileToolbar = () => {
   const { session } = useSession();
   const impersonation = useContext(ImpersonationContext);
   const [profileAnchor, setProfileAnchor] = useState<MenuProps['anchorEl']>();
-  const [actionsAnchor, setActionsAnchor] = useState<MenuProps['anchorEl']>();
 
   return (
     <>
-      <Card className={classes.card}>
+      <div className={classes.card}>
```

Comment:
I'd change this to a horizontal stack spacing 1.

Reply:
<Stack direction="row" spacing={1}>

yields

![Screenshot 2024-01-08 at 12 44 42](https://github.com/SeedCompany/cord-field/assets/1794353/89f7d8ea-2e55-4491-9a7c-0a4b4a8aecd8)

Reply:
Figured it out